### ISSUE TITLE
Why did I not merge to the master/main? It also shows that the branches have been merged

### ISSUE THREAD

**Issue by coverthesea** [go-gitea/gitea#22116]
### Feature Description

Only after I merge to main should it be displayed that I have merged
Now I just merge to the test branch, which also shows that the branch has been merged
Is this a different design?

**Comment by lunny** [comment]
But like Gitea has `main`, `release/**` may contains some commits from some branch. We don't know which branches should we stick.

**Comment by coverthesea** [comment]
> 

I think it is generally not the default branch
Similar to gitlab
I use gitlab now. Merged will be displayed only when merging to the main branch
I think it should be the default branch set

**Comment by wxiaoguang** [comment]
I didn't get the point about why "Merged should only be displayed when merging to the main branch".

What benefits does it bring?

Without some real cases, I also think current Gitea's approach is good enough: merged means that the new branch has been merged into another branch.

**Comment by GiteaBot** [comment]
We close issues that need feedback from the author if there were no new comments for a month. :tea:
